src/components/age.cairo

```diff
@@ -25,20 +25,16 @@ impl AgeImpl of AgeTrait {
 #[test]
 #[available_gas(30000000)]
 fn test_get_age_difference() {
-    let age = Age {
-        born_at: 100,
-    };
+    let age = Age { born_at: 100,  };
     let age_difference = age.get_age_difference(200);
     assert(age_difference == 100, 'Age difference should be 100');
 }
 
 #[test]
 #[available_gas(30000000)]
 fn get_current_age() {
```

Comment:
Unrelated to this PR, but the test function names should be prefixed with `test_`